code/modules/mob/living/carbon/human/combat.dm

```diff
@@ -26,19 +26,23 @@
 			var/turf/target = pick(turfs)
 			return G.afterattack(target, src, "struggle" = 1)
 
+	return 0
```

Comment:
If this is intended as a boolean, `return FALSE` would be more appropriate, in terms of allowing other people to easily understand its function.